main.go

```diff
@@ -81,8 +81,8 @@ func start(port string) {
 	state.SetClusterUuid(clusterUuid)
 	state.SetK8sVersion(k8sVersion)
 
-	server.InitServerVars()
-	certPath, keyPath, err := server.ValidateCertificate()
+	skipList.InitSkipList()
+	certPath, keyPath, err := skipList.ValidateCertificate()
```

Comment:
`ValidateCertificate()` has 0 connection to skipList. It lives w/o it

Reply:
agreed, and reverted to the previous code. in theory, they should get split into 2 separate packages